website/src/pages/index.tsx

```diff
@@ -22,59 +22,59 @@ type FeatureItem = {
 
 const FeatureList: FeatureItem[] = [
   {
-    title: "Pub/Sub logging",
+    title: "Pub/sub logging",
```

Comment:
I intentionally had this one capitalized because "Pub/Sub" as a concept seems to be commonly either both capitalized or both lowercase, even when used in sentence case phrases. e.g.

- https://redis.io/docs/manual/pubsub/
- https://support.google.com/cloud/answer/6255053?hl=en (granted this one is a product name)

If you want to make them all title case I'm fine with that too. I definitely think we should keep title case on the navbar and buttons.

Reply:
https://github.com/foxglove/mcap/pull/911